lib/membrane/rtp/twcc_sender.ex

```diff
@@ -32,9 +32,25 @@ defmodule Membrane.RTP.TWCCSender do
   end
 
   @impl true
-  def handle_pad_added(pad, _ctx, state) do
+  def handle_pad_added(pad, ctx, state) do
```

Comment:
Oh, w8, one more thing - we should only invoke this when new input pad is added, otherwise we will reset `cc` twice